cmd/sql-main.go

```diff
@@ -59,6 +81,45 @@ FLAGS:
 ENVIRONMENT VARIABLES:
    MC_ENCRYPT_KEY:  list of comma delimited prefix=secret values
 
+INPUT SERIALIZATION
+	--csv-input or --json-input can be used to specify input data format. Format is specified by a string
```

Comment:
This is really big perhaps for showing in `mc sql` we may trim it? perhaps point to the complete-guide?

Reply:
done.